district42/types/_str_schema.py

```diff
@@ -2,6 +2,7 @@
 from typing import Any
 
 from niltype import Nil, Nilable
+from typing_extensions import TypeAlias
```

Comment:
Please note that `typing_extensions` is not part of the standard Python distribution nor is it listed in the requirements for the current package. As a result, importing it will raise a `ModuleNotFoundError: No module named 'typing_extensions'`.

To avoid this issue, you can use one of the following approaches:

1. **Class-based conditional import**:

    ```python
    class StrSchema(Schema[StrProps]):

        if sys.version_info >= (3, 10):
            from typing import TypeAlias
            type: TypeAlias = str
        else:
            type: Any = str
    ```

2. **Global try-except import**:

    ```python
    try:
        from typing import TypeAlias
    except ImportError:
        TypeAlias = Any
    ```

Reply:
First option will not be working:
```
schemas/auth/auth_user.pyi:313: error: Variable "district42.types._str_schema.StrSchema.type" is not valid as a type  [valid-type]
schemas/auth/auth_user.pyi:313: note: See https://mypy.readthedocs.io/en/stable/common_issues.html#variables-vs-type-aliases

```